pkg/asset/manifests/imagedigestmirrorset.go

```diff
@@ -0,0 +1,89 @@
+package manifests
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	apicfgv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+)
+
+var imageDigestMirrorSetFilenameFormat = "image-digest-mirror-set-%s.yaml"
+
+// ImageDigestMirrorSet generates the image-digest-mirror-set.yaml files.
+type ImageDigestMirrorSet struct {
+	FileList []*asset.File
+}
+
+var _ asset.WritableAsset = (*ImageDigestMirrorSet)(nil)
+
+// Name returns a human-friendly name for the asset.
+func (*ImageDigestMirrorSet) Name() string {
+	return "Image Digest Mirror Set"
+}
+
+// Dependencies returns all of the dependencies directly needed to generate
+// the asset.
+func (*ImageDigestMirrorSet) Dependencies() []asset.Asset {
+	return []asset.Asset{
+		&installconfig.InstallConfig{},
+	}
+}
+
+// Generate generates the ImageDigestMirrorSet config and its CRD.
```

Comment:
```suggestion
// Generate generates the ImageDigestMirrorSet config and its CR.
```

?  you're generating a resource instance, not a resource definition, right?